docker-for-mac/osxfs.md

```diff
@@ -38,10 +38,19 @@ Docker user who started the containers.
 ### Namespaces
 
 Much of the OS X file system that is accessible to the user is also available to
-containers using the `-v` bind mount syntax. By default, you can share files in
-`/Users`, `/Volumes`, `/private`, and `/tmp` directly. To add or remove
-directory trees that are exported to Docker, use the **File sharing** tab in
-Docker preferences <img src="../images/whale-x.png"> -> **Preferences** ->
+containers using the `-v` bind mount syntax. For example, the command 
+
+`docker run -it -v ~/Desktop:/Desktop r-base bash`
+
+runs the Docker image called `r-base`, shares the Desktop of OSX in `/Desktop` and 
+starts bash. You can see the folder called Desktop inside the container such that
```

Comment:
starts `bash`.

s/such that/using the `ls`  command:

Also consider using `ls -c1` so that the listing will not be so wide. :) Then you can also truncate the listing maybe after `home` and use `...` to indicate that you truncated it.